azuredisk-csi.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: azuredisk-csi
+  version: 1.31.0
+  epoch: 0
+  description: Azure Disk CSI Driver
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 81f73ae6e758c2dd5efbaf127014e80d01d39f66
+      repository: https://github.com/kubernetes-sigs/azuredisk-csi-driver
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      ldflags: |
+        -X sigs.k8s.io/azuredisk-csi-driver/pkg/azuredisk.driverVersion=v${{package.version}}
+        -X sigs.k8s.io/azuredisk-csi-driver/pkg/azuredisk.gitCommit=$(git rev-parse HEAD)
+        -X sigs.k8s.io/azuredisk-csi-driver/pkg/azuredisk.buildDate=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+      output: azurediskplugin
+      tags: azurediskv2
+      packages: ./pkg/azurediskplugin
+
+subpackages:
+  - name: "${{package.name}}-compat"
+    description: "Compatibility package to place binaries in the location expected by upstream helm charts"
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"
+          ln -sf /usr/bin/azurediskplugin ${{targets.subpkgdir}}/azurediskplugin
+
+update:
+  enabled: true
+  github:
+    identifier: kubernetes-sigs/azuredisk-csi-driver
+    strip-prefix: v
+
+test:
+  pipeline:
+    - runs: |
+        # Run the azurediskplugin binary in the background
+        /usr/bin/azurediskplugin &
```

Comment:
We have a test pipeline, `test/daemon-check-output`, that we could likely use here if we get anything useful output in logs

Reply:
Thanks, RJ, used it, another TIL